tests/conftest.py

```diff
@@ -1138,6 +1138,14 @@ def disable_container_autorestart(duthost, testcase="", feature_list=None):
         # Dump autorestart state to file
         with open(state_file_name, "w") as f:
             json.dump(container_autorestart_states, f)
+        # For multi-asic duthost make sure the container
```

Comment:
The command `config feature autorestart {feature} disabled` should disable the feature on the host and in the asic namespaces if it present.
Do we know why the CLI command is not working

Reply:
@arlakshm,
#https://github.com/sonic-net/sonic-utilities/blob/f6b9912cfbefcabdbab3b32a538bcf354f11e204/config/feature.py#L142
Before disable/enable there is a check to verify the autorestart  state  across all config db instances. If the state is different across asics and host above error is seen.